build-tools/update

```diff
@@ -37,10 +37,19 @@ try {
 } catch (e) {}
 
 let configBuf = fs.readFileSync('config/config.js', {encoding: 'utf8'});
-configBuf = configBuf.replace(/\/\* version \*\/[^;\n]*;/, `/* version */ Config.version = ${JSON.stringify(version)};`);
+const routes = JSON.parse(fs.readFileSync('config/routes.json'));
+configBuf = configBuf.replace(/\/\* version \*\/[^;\n]*;/, `/* version */ Config.version = ${JSON.stringify(version)};`)
```

Comment:
I think we should probably have a clearly marked autogenerated section of Config. This approach might be a bit unclear as to what is going on.

Reply:
Sure, I took a stab at implementing that (my apologies if I missed something, it's 1 am right now).